torchtune/utils/_checkpointing/_checkpointer.py

```diff
@@ -431,6 +439,29 @@ def save_checkpoint(
             dim=self._config["hidden_size"],
         )
 
+        print(f"======={self._model_type}==========")
+        if (
+            self._model_type == "GEMMA"
```

Comment:
Can we move this to a separate utility function in checkpointer_utils? we should keep the checkpointer as clean as possible.

Reply:
Yeah, I did it in the latest version.